### PR TITLE
created epeen trigger

### PR DESCRIPTION
**Pull request by TheRealSuser**
added epeen.go

### CODE REVIEW

**Comment by bbriggs** [comment]
Add it to the default trigger map in `cmd/root.go` and we're good to go

File: cmd/root.go [hunk 1]
```diff
@@ -6,7 +6,7 @@
 // to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 // copies of the Software, and to permit persons to whom the Software is
 // furnished to do so, subject to the following conditions:
-//
+//				suser was here
```
Comment (m-242) [inline cmd/root.go]: lolnope

**Comment by m-242** [comment]
Changed the destination branch, so that we follow the git flow we talked about.

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
    - Config struct updated to include plugins and a mutex
    - Plugin map type is now `map[string]NamedTrigger` instead of `sync.Map` and guarded with a `sync.RWMutexv`
+## [1.1.0] - 2019-12-16
+### Added
+   - added epeen function to display the requesters peepee
```
Comment (bbriggs) [inline CHANGELOG.md]: You only need to add a summary of the overall feature, not each individual file and commit

File: CHANGELOG.md [hunk 3]
```diff
@@ -36,6 +36,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+   - epeen.go
+   - added the trigger to root.go
+   -guys...suser was also here
```
Comment (bbriggs) [inline CHANGELOG.md]: Add a space if you want this to format properly

**Pull request merged**